### PR TITLE
4.1.2 task solutions comments files seeds kyselov

### PR DESCRIPTION
**Pull request by IselDrew**
Add seeds to tasks, solutions, comments and files

### CODE REVIEW

File: data/comments.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "solutionId": "1",
```
Comment (Alan18081) [inline data/comments.json:3]: It is better to have it as number

File: src/migrations/20200426014953-addColumn-complete-solutionTable.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { QueryInterface, DataTypes } from "sequelize";
+import { Solution } from "../modules/solutions/solutions.model";
+
+export async function up(query: QueryInterface) {
+    return query.addColumn(Solution.tableName, "isCompleted", {
+        type: DataTypes.INTEGER,
+        allowNull: true,
+        onUpdate: "CASCADE",
```
Comment (Alan18081) [inline src/migrations/20200426014953-addColumn-complete-solutionTable.ts:8]: Use single-quotes

File: src/modules/tasks/tasks.dtos.ts [hunk 3]
```diff
@@ -3,11 +3,11 @@ import * as Joi from 'joi';
 export const createTaskDto = Joi.object().keys({
   groupId: Joi.number().required(),
   taskName: Joi.string().min(2).required(),
-  fileURL: Joi.string().uri(),
+  description: Joi.string().optional(),
```
Comment (Alan18081) [inline src/modules/tasks/tasks.dtos.ts:6]: Do we really need `optional`?

File: src/modules/tasks/tasks.dtos.ts [hunk 4]
```diff
@@ -3,11 +3,11 @@ import * as Joi from 'joi';
...
 });
 
 export const updateTaskDto = Joi.object().keys({
   groupId: Joi.number(),
   taskName: Joi.string().min(2),
-  fileURL: Joi.string().uri(),
+  description: Joi.string().optional(),
```
Comment (Alan18081) [inline src/modules/tasks/tasks.dtos.ts:12]: Do we really need `optional`?

File: data/comments.json (hunk 1 above)
Comment (IselDrew) [reply data/comments.json]: fixed

File: src/migrations/20200426014953-addColumn-complete-solutionTable.ts (hunk 2 above)
Comment (IselDrew) [reply src/migrations/20200426014953-addColumn-complete-solutionTable.ts]: fixed

File: src/modules/tasks/tasks.dtos.ts (hunk 3 above)
Comment (IselDrew) [reply src/modules/tasks/tasks.dtos.ts]: fixed

File: src/migrations/20200426014953-addColumn-complete-solutionTable.ts [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import { QueryInterface, DataTypes } from "sequelize";
+import { Solution } from "../modules/solutions/solutions.model";
+
```
[the remark this answers is not in the document]
Comment (IselDrew) [reply src/migrations/20200426014953-addColumn-complete-solutionTable.ts]: fixed

File: data/comments.json [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    "text": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua"
+  },
+  {
+    "solutionId": "1",
+    "studentId": null,
+    "teacherId": "1",
+    "text": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua"
+  },
+  {
+    "solutionId": "1",
```
[the remark this answers is not in the document]
Comment (IselDrew) [reply data/comments.json]: fixed

File: data/solutions.json [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+    "taskId": "1",
+    "grade": null,
+    "isCompleted": 0 
+  },
+  {
+    "studentId": "10",
+    "taskId": "1",
+    "grade": null,
+    "isCompleted": 0 
+  }
```
[the remark this answers is not in the document]
Comment (IselDrew) [reply data/solutions.json:253]: fixed

**Comment by IselDrew** [comment]
Change fileNameExtension and add also add avatar seeds

File: src/modules/tasks/tasks.dtos.ts (hunk 4 above)
Comment (IselDrew) [reply src/modules/tasks/tasks.dtos.ts]: fixed

File: src/modules/tasks/tasks.dtos.ts (hunk 3 above)
Comment (IselDrew) [reply src/modules/tasks/tasks.dtos.ts]: fixed

**Pull request merged**